controller/src/main/java/org/jboss/as/controller/AbstractAttributeDefinitionBuilder.java

```diff
@@ -414,6 +414,16 @@ public BUILDER setStorageRuntime() {
     }
 
     /**
+     * Adds the {@link AttributeAccess.Flag#STORAGE_RUNTIME} flag and removes any conflicting flag.
+     *
+     * @return a builder that can be used to continue building the attribute definition
+     */
+    public BUILDER setStorageRuntime(boolean forceRegistration) {
```

Comment:
Actually, I am afraid I don't really like this parameter either. The reason is that .setStorageRuntime(false) will make it look like it turns it off. Can we just keep the original method and have an additional .forceRegistration() method or something like that instead?